packages/autocomplete-js/src/getDefaultOptions.ts

```diff
@@ -101,15 +101,6 @@ export function getDefaultOptions<TItem extends BaseItem>(
     'The `container` option does not support `input` elements. You need to change the container to a `div`.'
   );
 
-  warn(
-    !(render && renderer && !renderer?.render),
```

Comment:
what if we pass an empty function in the example? that way it's still a good warning for other cases

Reply:
I like this idea 👍 Changed in [54e6718](https://github.com/algolia/autocomplete/pull/969/commits/54e6718135f26a8a78d70f548a6b37097ac11644) and amended the message in the warning for React 18 users.